curvefs/src/metaserver/copyset/copyset_service.cpp

```diff
@@ -79,13 +79,17 @@ void CopysetServiceImpl::GetCopysetsStatus(
 
 COPYSET_OP_STATUS CopysetServiceImpl::CreateOneCopyset(
     const CreateCopysetRequest::Copyset& copyset) {
-    bool exists =
-        manager_->IsCopysetNodeExist(copyset.poolid(), copyset.copysetid());
-    if (exists) {
+    int exists = manager_->IsCopysetNodeExist(copyset);
```

Comment:
another problem is above `CreateCopysetNode` will stop directly when failed to create one copyset, so you can find whether there's a good reason to continue creating all copysets and return status for each copyset.